R/asis-nodes.R

```diff
@@ -107,10 +107,7 @@ protect_inline_math <- function(body, ns) {
 
   # protect math that is strictly inline
   if (length(imath)) {
-    new_nodes <- lapply(
-      fix_fully_inline(imath), 
-      FUN = function(n) xml2::xml_ns_strip(xml2::xml_children(n))
-    )
+    new_nodes <- lapply(imath, fix_fully_inline)
```

Comment:
given the package depends on purrr, should we use it here?

Reply:
That's a good point for consistency's sake!